source/B2BApi.AppTests/Fixtures/B2BApiAppFixture.cs

```diff
@@ -179,15 +183,26 @@ public async Task InitializeAsync()
         LogStopwatch(stopwatch, nameof(CreateAppHostSettings));
 
         // ServiceBus entities
-        TopicResource = await ServiceBusResourceProvider
+        IntegrationEventsTopicResource = await ServiceBusResourceProvider
             .BuildTopic("integration-events")
             .Do(topic => appHostSettings.ProcessEnvironmentVariables
                 .Add($"{IntegrationEventsOptions.SectionName}__{nameof(IntegrationEventsOptions.TopicName)}", topic.Name))
             .AddSubscription("subscription")
             .Do(subscription => appHostSettings.ProcessEnvironmentVariables
                 .Add($"{IntegrationEventsOptions.SectionName}__{nameof(IntegrationEventsOptions.SubscriptionName)}", subscription.SubscriptionName))
             .CreateAsync();
-        LogStopwatch(stopwatch, nameof(TopicResource));
+        LogStopwatch(stopwatch, nameof(IntegrationEventsTopicResource));
+
+        var processManagerTopic = await ServiceBusResourceProvider
```

Comment:
Maybe I miss something in the PR, but I would expect the following:
```suggestion
        ProcessManagerTopicResource  = await ServiceBusResourceProvider
```

Reply:
Fixed